packages/cfd/src/Containers/props.types.ts

```diff
@@ -241,33 +241,45 @@ type TOpenAccountTransferMeta = {
     type?: string;
 };
 
-export type TJurisdictionModalProps = {
+export type TJurisdictionModalContentWrapperProps = {
```

Comment:
since TJurisdictionModalContentWrapperProps has all the props in TJurisdictionModalProps, cant we use it as below?

export type TJurisdictionModalContentWrapperProps =TJurisdictionModalProps & {
....   // remaining props
}

Reply:
@amina-deriv 2 properties 'enableApp' and 'disableApp' are extra in TJurisdictionModalProps. when we extend it, code editors may display these in suggestions to developers. is it ok, shall i write like this?

Reply:
Cant we extract the common props out and declare it as separate type to reuse it?